subliminal/providers/itasa.py

```diff
@@ -0,0 +1,390 @@
+        while next.text is not None:
+
+            r = self.session.get(next.text, timeout=10)
+            r.raise_for_status()
+            root = etree.fromstring(r.content)
+
+            logger.info('Loading suggestion page %s', root.find('data/page').text)
+
+            # Looking for show in following pages
+            for show in root.findall('data/shows/show'):
+                if sanitize(show.find('name').text).lower() == sanitize(series.lower()):
+                    show_id = int(show.find('id').text)
+                    logger.debug('Found show id %d', show_id)
+
+                    return show_id
+
+            next = root.find('data/next')
+
+        # No matches found
+        logger.warning('Show id not found: suggestions does not match')
+
+        return None
+
+    def get_show_id(self, series, country_code=None):
+        """Get the best matching show id for `series`.
+
+        First search in the result of :meth:`_get_show_ids` and fallback on a search with :meth:`_search_show_id`
+
+        :param str series: series of the episode.
+        :return: the show id, if found.
+        :rtype: int or None
+
+        """
+        series_sanitized = sanitize(series).lower()
+        show_ids = self._get_show_ids()
+        show_id = None
+
+        # attempt with country
+        if not show_id and country_code:
+            logger.debug('Getting show id with country')
+            show_id = show_ids.get('%s %s' % (series_sanitized, country_code.lower()))
+
+        # attempt clean
+        if not show_id:
+            logger.debug('Getting show id')
+            show_id = show_ids.get(series_sanitized)
+
+        # search as last resort
+        if not show_id:
+            logger.warning('Series not found in show ids')
+            show_id = self._search_show_id(series)
+
+        return show_id
+
+    def _download_zip(self, sub_id):
+        # download the subtitle
+        logger.info('Downloading subtitle %r', sub_id)
+
+        params = {
+            'authcode': self.auth_code,
+            'apikey': self.apikey,
+            'subtitle_id': sub_id
+        }
+
+        r = self.session.get(self.server_url + 'subtitles/download', params=params, timeout=30)
+        r.raise_for_status()
+
+        return r.content
+
+    def query(self, series, season, episode, format, resolution, country=None, hash=None):
+
+        # To make queries you need to be logged in
+        if not self.logged_in:
+            raise ConfigurationError('Cannot query if not logged in')
+
+        # get the show id
+        show_id = self.get_show_id(series, country)
+        if show_id is None:
+            logger.error('No show id found for %r ', series)
+            return []
+
+        # get the page of the season of the show
+        logger.info('Getting the subtitle of show id %d, season %d episode %d, format %s', show_id,
+                    season, episode, format)
+        subtitles = []
+
+        # Default format is HDTV
```

Comment:
My bad here: 
Default format should be 'normale' == 'sdtv'. If I not specify the format, then this will retrieve all possible subtitles for episode.
I had choosen a reasonable default format, so if no data come from video object, it will download a reasonable subtitle.